testing/README.md

```diff
@@ -0,0 +1,95 @@
+  - `export AWS_SECRET_ACCESS_KEY=$(echo $output | jq -r .Credentials.SecretAccessKey)`
+  - `export AWS_SESSION_TOKEN=$(echo $output | jq -r .Credentials.SessionToken)`
+- Run `aws sts get-caller-identity` to check if you are in the correct role
+#### Create a new Cluster
+- Next, create the cluster by running `eksctl create cluster --name <ClusterName> --region us-east-1 --zones us-east-1a,us-east-1b`. This will take around ~10 minutes. 
+#### Install AWS Load Balancer Controller Add-on
+- Finally, install the AWS Load Balancer Controller add-on by running the following commands. Make sure to replace the `<ClusterName>` and `<AccountID>` with the correct value.
+  - `eksctl utils associate-iam-oidc-provider --cluster <ClusterName> --region us-east-1 --approve`
+  - `curl -O https://raw.githubusercontent.com/kubernetes-sigs/aws-load-balancer-controller/v2.4.7/docs/install/iam_policy.json`
+  - `aws iam create-policy --policy-name AWSLoadBalancerControllerIAMPolicy --policy-document file://iam_policy.json --region us-east-1`
+  - `eksctl create iamserviceaccount --cluster=<ClusterName> --namespace=kube-system --name=aws-load-balancer-controller --attach-policy-arn=arn:aws:iam::<AccountID>:policy/AWSLoadBalancerControllerIAMPolicy --region us-east-1 --approve`
+  - `kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v1.5.4/cert-manager.yaml`
+  - `curl -Lo v2_4_7_full.yaml https://github.com/kubernetes-sigs/aws-load-balancer-controller/releases/download/v2.4.7/v2_4_7_full.yaml`
+  - `sed -i.bak -e '561,569d' ./v2_4_7_full.yaml`
+  - `sed -i.bak -e 's|your-cluster-name|<ClusterName>|' ./v2_4_7_full.yaml`
+  - `kubectl apply -f v2_4_7_full.yaml`
+  - `curl -Lo v2_4_7_ingclass.yaml https://github.com/kubernetes-sigs/aws-load-balancer-controller/releases/download/v2.4.7/v2_4_7_ingclass.yaml`
+  - `kubectl apply -f v2_4_7_ingclass.yaml`
+
+### 3. Setting up Environment for EC2 Tests
+#### Create IAM Role for EC2 Instance
+- Login to AWS, go to the IAM console and click on the Roles tab. Click Create role.
+- Choose AWS service, and choose EC2 as the use case. Click Next.
+- Choose AmazonS3ReadOnlyAccess, AWSXrayWriteOnlyAccess, and CloudWatchAgentServerPolicy as the permission. 
+- Type the role name and click "Create role".
+- 
+#### Setting Up Default VPC
+- Go to the VPC console and on the routing table for the default VPC, click Edit routes. 
+- Click add routes, for destination add `0.0.0.0/0`, for target add Internet Gateway and save changes.
+- Go to the Security groups tab, find the security group attached to the default VPC, click Edit inbound rules, choose type: All Traffic, Source: custom, and CIDR block: 0.0.0.0/0. Save rules.
+
+### 4. Building Sample App to ECR
+Create two ECR repositories: one for the sample app main service and another for the sample app remote service. 
+Follow the instructions under ./sample-apps/README.md to build the sample app image and upload it to the ECR
```

Comment:
IIRC there's a way to link between pages instead of giving the path

Reply:
Done